### PR TITLE
phonePrefix and time localisation

### PR DESCRIPTION
**Pull request by gabrielonilsson**

### CODE REVIEW

File: DateTime.ts [hunk 1]
```diff
@@ -15,4 +15,12 @@ export namespace DateTime {
 	export function now(): DateTime {
 		return create(new Date())
 	}
+	export function localize(value: Date, locale?: string, options?: object): DateTime {
```
Comment (simonmika) [inline DateTime.ts]: Remove `options`. Be more opinionated and don't leak the underlying implementation, abstract instead.

Comment (simonmika) [inline DateTime.ts]: We should create a `Locale` type in isoly, or? @fredrikbryntesson, what do you think?

File: DateTime.spec.ts [hunk 2]
```diff
@@ -6,4 +6,24 @@ describe("DateTime", () => {
 		expect(model.DateTime.is(d))
 		expect(d).toBe("2020-12-31T23:59:59.000Z")
 	})
+	it("localize with arguments", () => {
+		const date = new Date("20 Jul 2019 10:30:40 GMT+2")
```
Comment (simonmika) [inline DateTime.spec.ts]: It is nicer to store them in an array. That way adding more is easier. Do it like this:
```
const data = [
 [<value0>, <expected0>],
 [<value1>, <expected1>],
]
```
and then loop over the array and to the test in stead of echoing yourself.

**Review by simonmika — CHANGES_REQUESTED** [review]

File: DateTime.ts (hunk 1 above)
Comment (gabrielonilsson) [reply DateTime.ts]: This remains to be done.

**Review by simonmika — APPROVED** [review]

**Pull request merged**